src/NuGet.Core/NuGet.Configuration/PackageSource/PackageSourceUpdateSettings.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Configuration
+{
+    public sealed class PackageSourceUpdateSettings
+    {
+        public readonly static PackageSourceUpdateSettings Default = new PackageSourceUpdateSettings(true, true);
+
+        public bool UpdateCredentials { get; private set; }
+        public bool UpdateEnabled { get; private set; }
```

Comment:
nitpick:

```suggestion
        public bool UpdateCredentials { get; }
        public bool UpdateEnabled { get; }
```

This only lets the ctor set the value, not other methods in the class (although this class doesn't have any other method).